### ISSUE TITLE
Do you plan to use ORM? 

### ISSUE THREAD

**Issue by sagruob** [ccwukong/ElfCommerce#68]
I've test your project out there. I think ORM integration it will be a great benefit in future. 

btw I've tried to integrate sequelize in your project and it seems pretty straightforward and save a lot of redundant code. Still needs some tweaking to make it 100% working tho.

**Comment by ccwukong** [comment]
Yep, sequelize is handy and pls help us to implement it if you'd like to be part of this project. thanks
